### ISSUE TITLE
Missing untagged Vim patches not reported by vim-patch.sh

### ISSUE THREAD

**Issue by jusga** [neovim/neovim#3964]
Currently `vim-patch.sh -l` assumes that all untagged Vim patches before and including vim/vim@e2719096250a19ecdd9a35d13702879f163d2a50 have been merged. However this is not entirely true:

vim/vim@e2719096250a19ecdd9a35d13702879f163d2a50 is missing
vim/vim@8feef4ff6253afb9dcc61c40082ed4fbb96b685c was merged (d69f3bbb0d3a0549466ac0e36a1364d1890b4691)
~~vim/vim@e3faf44bef029d07f37a457bd0050653b628058f is missing~~ #3965
vim/vim@ddfb92548044d27318294c1c931aab2188580546 was merged (183b4b60b68d147b5b392114d9ed582d0f490f9f, old hg hash in commit title)
vim/vim@ed32d9424d12f93dcdb96f2a5c512f7ad9945bee was merged (63d5b8707b3b35b91b4dd23d578e5b0adfb2a9b2, old hg hash in commit title)
vim/vim@2a0b33e2841ad0c23f4385269f98701d1b105107 was merged (86330fdd3f39a8975b25b7050a3d02b77a442057)
vim/vim@2f3b510b0f9203d31e2f7f536ebfbcb17766e09e was merged (04a3d5ecc7c9f626b28f60fdd76f08e0e9c867d3, old hg hash in commit title)

This is getting tedious, so I will stop here and hope that all older patches have been merged.

I will open PRs for the two missing patches shortly.

**Comment by justinmk** [comment]
@jusga Thank you for that attention to detail. It's bizarre that those fell through the cracks but yes, I think now we can be pretty sure that we found that last stragglers. My original investigation which drew the line at https://github.com/vim/vim/commit/e2719096250a19ecdd9a35d13702879f163d2a50 was perhaps confused by the fact that people had send out-of-order patches and I guess I didn't look closely enough (though I did try).
